### PR TITLE
shinydashboardPlus removal

### PR DESCRIPTION
**Pull request by riccardoporreca**
Merge of #113 (see details there).

### CODE REVIEW

File: inst/application/ui.R [hunk 1]
```diff
@@ -5,30 +5,35 @@
 
 Sys.setlocale("LC_TIME", "C")
 
-bs_embed_tooltip <- bsplus::bs_embed_tooltip # info windows
-boxPlus <- shinydashboardPlus::boxPlus
+# info tooltips
+bs_embed_tooltip_body <- function(..., container = "body") {
+  # Specify container = "body", which is particularly useful to get widr tootips
```
Comment (nfarabullini) [inline inst/application/ui.R]:
```suggestion
  # Specify container = "body", which is particularly useful to get wider tooltips
```

File: inst/application/ui.R [hunk 2]
```diff
@@ -278,21 +264,21 @@ fluidPage(
...
                              step = 0.1,
                              min = 0
                 ) %>%
-                  bs_embed_tooltip(title = IB$returnP3, placement = "right")
+                  bs_embed_tooltip_body(title = IB$returnP3, placement = "right")
               )
             )
-          ) # end of 3rd Pillar boxPlus
-        ), # end of 3rd Pillar fluidRow
+          ), # end Pillar III SmaRPanel
```
Comment (riccardoporreca) [inline inst/application/ui.R]: We should align this comment to the new pillar naming, and also align the `id` of the panels above.

Comment (riccardoporreca) [reply inst/application/ui.R]: @RolandASc, I just realized I never pushed some updates I committed locally after merging the latest updates from develop back in May! I am fixing the mess!

Comment (riccardoporreca) [reply inst/application/ui.R]: Fixed un-pushed conflicts in  8965b42

Comment (riccardoporreca) [reply inst/application/ui.R]: Fixed panel ids in 
7f12f6f

**Comment by gabrielfoix** [comment]
Looks fine to me. 
I don't see any change on the UI, looks exactly the same.

**Pull request merged**